functions/compare_classification_facts.pp

```diff
@@ -0,0 +1,39 @@
+# @summary Compare untrusted values from the classification fact to their trusted values.
+#
+# Compare untrusted values from the classification fact to their trusted values.
```

Comment:
Unless it makes a difference in the strings output, I'd just remove this line since it's the same as the @summary.

Reply:
It makes a difference (sadly)

Reply:
It's used by `puppet strings`

Reply:
😭